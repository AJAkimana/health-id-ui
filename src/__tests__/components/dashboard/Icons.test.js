import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SVGIcon from '../../../components/shared/Dashboard/Icons';

const props = {
  name: '',
  style: {},
  fill: ''
}

describe('SVGIcon', () => {
  it('should render the SVGIcon component properly', () => {
    const setupSVGIcon = () => {
      const wrapper = shallow(<SVGIcon {...props} />)
      return wrapper
    }

    let wrapper;
    wrapper = setupSVGIcon();
    expect(toJson(wrapper)).toMatchSnapshot();
  })
});
