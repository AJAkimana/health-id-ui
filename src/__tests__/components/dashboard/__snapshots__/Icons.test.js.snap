// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SVGIcon should render the SVGIcon component properly 1`] = `
<svg
  name=""
  style={Object {}}
  viewBox="0 0 0.268 24.039"
  xmlns="http://www.w3.org/2000/svg"
>
  <path />
</svg>
`;
